P1/Project Brief/entertainment-web-app/server/src/routes/shows.router.js

```diff
@@ -0,0 +1,15 @@
+import { Router } from 'express';
+import { getShows , postShow } from "../controllers/show.controller.js" ;
+import { upload } from '../middlewares/multer.middleware.js';
+
+const router = Router();
+
+router.route('/shows').get( 
```

Comment:
assumming the route will be defined as "api/v1/shows" in app.js then you should simply this route to "/"

Reply:
> assumming the route will be defined as "api/v1/shows" in app.js then you should simply this route to "/"

Okay I will make the said changes